lib/adal-angular.js

```diff
@@ -344,9 +342,22 @@
                         else {
                             // Cancel request if login is starting
                             if (authService.loginInProgress()) {
-                                authService.info('login is in progress.');
-                                config.data = 'login in progress, cancelling the request for ' + config.url;
-                                return $q.reject(config);
+                                if (authService.config.popUp) {
+                                    authService.info('Url: ' + config.url + ' will be loaded after login is successful');
+                                    var delayedRequest = $q.defer();
+                                    $rootScope.$on('adal:loginSuccess', function (event, token) {
+                                        if (token) {
+                                            config.headers.Authorization = 'Bearer ' + tokenStored;
```

Comment:
Add a log statement here too: "Login Completed. Sending request for " + config.url.